docs/src/pages/blog/introducing-anywidget.mdx

```diff
@@ -392,7 +392,7 @@ modern front-end development tools. The [Vite plugin](/en/bundling#development)
 for **anywidget** extends [Vite's](https://vitejs.dev/) development server with precise
 [Hot Module Replacement](https://vitejs.dev/guide/features.html#hot-module-replacement) support for Jupyter Widgets.
 Changes made to the JavaScript source during development are **instantly** reflected
-in the front end without refreshing or re-executing notebook cells.
+in the front-end without refreshing or re-executing notebook cells.
```

Comment:
People seem to have differing opinions about frontend vs front-end vs front end. In this article I use "front-end" when it is used as a compound adjective and "front end" when used as a noun (a recommendation I've seen [several places](https://css-tricks.com/poll-results-front-end-front-end/#aa-the-right-answer-seems-to-be).

Reply:
Oh I didn't know :+1: